interactives/high-score-boxes/index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <!--Let browser know website is optimized for mobile-->
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+    <!-- ### Enter your interactive title in the title block ### -->
+    <title>High Score Boxes </title>
+    <!--<link rel="stylesheet" href="../base-files/css/third-party/normalize.css">-->
+    <!--Import materialize.css-->
+    <link rel="stylesheet" href="../base-files/css/third-party/materialize.min.css" type="text/css" media="screen,projection"/>
+    <script src="../base-files/js/third-party/modernizr.js"></script>
+    <!-- jQuery (necessary for Materialize) -->
+    <script type="text/javascript" src="../base-files/js/third-party/jquery.js"></script>
+  </head>
+  <body>
+    <div class="row">
+    <div class="col s12 m8 l6 offset-l3 offset-m2">
+    <!-- ### Start of required HTML for inline interactives ### -->
+    <div class="interactive" id="interactive-high-score-boxes">
+    <h1>High Score Boxes</h1>
+    <p>What is the largest number in the boxes? </p>
+    <div class="row">
+    	<div class="col s12">
+			<div id="box_holder_div">
+			</div>
+		</div>
+		<div class="col s12">
+			<form name="myForm"  action="congratulations.html"
+			onsubmit="return validateForm()" method="post" >
+            <input type="text" id="answerTextForm" placeholder="Enter answer here..." name="submittedInt">
+			<input type="submit"  value="Submit">
+			</form>
+            <p id="completionMessage"> </p>
+            <a id="restartButton" class="btn btn-large btn-expand" >Restart</a>
```

Comment:
This button is not rendering correctly as it's being displayed `inline` rather than `block`. Try changing the element from an `a` element to a `button` element. Also the `btn-expand` class is doing nothing.